src/api/controllers/commentController.js

```diff
@@ -0,0 +1,185 @@
+import _ from 'lodash';
+import Sequelize from 'sequelize';
+import status from '../../helpers/constants/status.codes';
+import models from '../models';
+import sendError from '../../helpers/error.sender';
+import errorMessages from '../../helpers/constants/error.messages';
+import generatePaginationDetails from '../../helpers/generate.pagination.details';
+
+const { Comment, User } = models;
+
+/**
+ *
+ *
+ * @class CommentController
+ */
+class CommentController {
+  /**
+   * A controller to post a new comment on an article
+   *
+   * @author Prémices
+   * @static
+   * @param {object} req - this is the request object
+   * @param {object} res - this is the response object
+   * @returns {object} - this function returns an object
+   * @memberof RatingsController
```

Comment:
@nkpremices edit the comment here `RatingsController`
Also, your comment here is confusing: `A controller to post a new comment on an article`
Is the comment referring to the entire controller or just a method? If it's referring to a method, then `A method to post a new comment on an article` is more appropriate.

Reply:
It's actually referring to just a method @MemunaHaruna 
let me go there and modify the comment

Reply:
Cool @nkpremices .. you forgot this `@memberof RatingsController`